TSOClient/tso.client/UI/Panels/UIUCP.cs

```diff
@@ -324,6 +325,22 @@ private void UpdateWallsViewCallback(int mode)
             Game.LotControl.WallsMode = mode;
             UpdateWallsMode();
         }
+        private void UpdateWallsViewKeyHandler(int type)
```

Comment:
Would probably be better passing a delta here, rather than an unnamed enum. Will fix on pull.